communication/parameterserver.h

```diff
@@ -0,0 +1,23 @@
+#ifndef PARAMETERSERVER_H
+#define PARAMETERSERVER_H
+
+#include <QObject>
+#include <mavsdk/server_component.h>
+#include <mavsdk/plugins/param_server/param_server.h>
+#include <unordered_map>
+
+class ParameterServer : public QObject
```

Comment:
Sooner or later we will need one more layer of abstraction. This implementation is MAVLINKParameterServer, because it  relies on MAVSDK. Once a class includes this, it will become MAVSDK-dependent as well. To avoid this, classes that want to provide parameters should be able to include and work on an abstract class ParameterServer that is protocol independent and allows implementation of different protocols.

Reply:
One more comment: it might make sense to make the abstract parameterServer a singleton (https://en.wikipedia.org/wiki/Singleton_pattern, https://stackoverflow.com/questions/1091314/singleton-and-abstract-base-class-in-c) which enable all classes that want to provide parameters to access the one right instance of the server by just including it.